README.md

```diff
@@ -104,17 +104,18 @@ print(f"Factuality metadata: {result.metadata['rationale']}")
 experiment = braintrust.init(
     project="Autoevals", api_key="YOUR_BRAINTRUST_API_KEY"
 )
-experiment.log(
-    inputs={"query": input},
-    output=output,
-    expected=expected,
-    scores={
-        "factuality": result.score,
-    },
-    metadata={
-        "factuality": result.metadata,
-    },
-)
+with experiment.start_span() as span:
```

Comment:
should we adjust it to call the `evaluator` in the span, so it gets traced properly?